tests/src/JIT/HardwareIntrinsics/X86/Shared/HorizontalBinOpTest.template

```diff
@@ -0,0 +1,336 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
New template. Used for tests which perform horizontal operations.

Reply:
👍 Thanks, I will port AVX horizontal intrinsic to this new template.